lxrhash_test.go

```diff
@@ -40,6 +40,28 @@ func BenchmarkHash(b *testing.B) {
 			lx.Hash(no)
 		}
 	})
+	b.Run("flat hash", func(b *testing.B) {
+		nonce := []byte{0, 0}
+		for i := 0; i < b.N; i++ {
+			nonce = nonce[:0]
+			for j := i; j > 0; j = j >> 8 {
+				nonce = append(nonce, byte(j))
+			}
+			no := append(oprhash, nonce...)
+			lx.Hash(no)
+		}
+	})
+	b.Run("flat hash again", func(b *testing.B) {
+		nonce := []byte{0, 0}
+		for i := 0; i < b.N; i++ {
+			nonce = nonce[:0]
+			for j := i; j > 0; j = j >> 8 {
+				nonce = append(nonce, byte(j))
+			}
+			no := append(oprhash, nonce...)
+			lx.Hash(no)
+		}
+	})
```

Comment:
You are using `Hash()` still here. Just change it to `FlatHash` and I'll approve

Reply:
done